frontend/src/components/samples/SamplesListContent.js

```diff
@@ -22,7 +22,15 @@ import FiltersWarning from "../filters/FiltersWarning";
 import SamplesFilters from "./SamplesFilters";
 import mergedListQueryParams from "../../utils/mergedListQueryParams";
 
-const getTableColumns = (containersByID, individualsByID, sampleKinds) => [
+const getProjectLinks = (ids, projectsByID) => {
+  return (
+    ids.map(id => {
+      return (<div> <Link to={`/projects/${projectsByID[id].id}`}> {projectsByID[id].name} </Link> </div>);
+    })
+  )
+}
```

Comment:
No need for a distinct function for that. This is used only to render the content of a cell in the list. The other cells are rendered directly in the render function of the appropriate getTableColumns column